watertap/unit_models/reverse_osmosis_1D.py

```diff
@@ -202,46 +208,15 @@ def build(self):
         # Add reference to pressure drop for feed side only
         if (self.config.has_pressure_change is True and
                 self.config.momentum_balance_type != MomentumBalanceType.none):
-            add_object_reference(self, 'deltaP', feed_side.deltaP)
+            add_object_reference(self, 'dP_dx', feed_side.deltaP)
 
         self._make_performance()
 
         self._add_expressions()
 
+        # TODO: why is this here?
         self._get_performance_contents()
```

Comment:
I don't think it needs to be there. This was one of my earliest PRs so it was probably an oversight.

Reply:
Removed.